Classes/Views/Constants.swift

```diff
@@ -41,5 +41,6 @@ enum Constants {
         static let delete = NSLocalizedString("Delete", comment: "")
         static let inbox = NSLocalizedString("Inbox", comment: "")
         static let upload = NSLocalizedString("Upload", comment: "")
+        static let bookmark = NSLocalizedString("Bookmark", comment: "")
```

Comment:
Is this used anywhere else yet? Rule of thumb is to only put something in `Constants.swift` when its used >1.